index.html

```diff
@@ -9,47 +9,157 @@
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj3.png" alt="project 3" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj4.png" alt="project 4" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj5.png" alt="project 5" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project   
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+  
+        <div class="proj-1">
+          <img src="static/Proj6.png" alt="project 6" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+      </section>
+  
+      <section class="about">
+        <h2 class="h2">About</h2>
+        <div class="about-description">
+          
+          <p>
+            I can help you build a product , feature or <br />
+            website Look through some of my work <br />
+            and experience! If you like what you see <br />
+            and have a project you need coded, <br />
+            don’t hesitate to contact me.
+          </p>
+          <button type="button">
+            Get my CV <img src="static/ic_download.svg" />
+          </button>
+        </div>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: this about section should have a left and right padding as seen in the Figma design. Kindly ensure you implement this. well done :clap: you did changes given by the previous reviewer but still, there are some issues there kindly check below :point_down: the comparison of your and Figma's design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-05 23-50-29](https://user-images.githubusercontent.com/45226848/236544991-3ab7d279-9e23-490f-ae88-15dbb7f1c0fa.png)

**Figma Design**
![236502126-ea3bf922-9a1b-4be2-98c1-53bcfc38437e](https://user-images.githubusercontent.com/45226848/236545070-b96db01b-5af8-44ba-a4a9-67e21d3ffd62.png)

- [ ] You can see in the Figma design, that there is a white space on the left and right sides, and the Get my Cv button spans the whole width of the screen.